imap_processing/cdf/config/imap_hit_l1a_variable_attrs.yaml

```diff
@@ -0,0 +1,500 @@
+  FORMAT: I3
+
+fee_running:
+  <<: *support_default
+  VALIDMAX: 1
+  CATDESC: State - FEE Running (1) or Reset (0)
+  FIELDNAM: FEE Running (1) or Reset (0)
+  LABLAXIS: State
+  FORMAT: I1
+
+mram_disabled:
+  <<: *support_default
+  VALIDMAX: 1
+  CATDESC: State - MRAM Disabled (1) or Enabled (0)
+  FIELDNAM: MRAM Disabled (1) or Enabled (0)
+  LABLAXIS: State
+  FORMAT: I1
+
+enable_50khz:
+  <<: *support_default
+  VALIDMAX: 1
+  CATDESC: State - 50kHz Enabled (1) or Disabled (0)
+  FIELDNAM: 50kHz Enabled (1) or Disabled (0)
+  LABLAXIS: State
+  FORMAT: I1
+
+enable_hvps:
+  <<: *support_default
+  VALIDMAX: 1
+  CATDESC: State - HVPS Enabled (1) or Disabled (0)
+  FIELDNAM: HVPS Enabled (1) or Disabled (0)
+  LABLAXIS: State
+  FORMAT: I1
+
+table_status:
+  <<: *support_default
+  VALIDMAX: 1
+  CATDESC: State - Table Status OK (1) or Error (0)
+  FIELDNAM: Table Status OK (1) or Error (0)
+  LABLAXIS: Status
+  FORMAT: I1
+
+heater_control:
+  <<: *support_default
+  VALIDMAX: 2
+  CATDESC: State - Heater Control (0=None, 1=Pri, 2=Sec)
+  FIELDNAM: Heater Control (0=None, 1=Pri, 2=Sec)
+  LABLAXIS: State
+  FORMAT: I1
+
+adc_mode:
+  <<: *support_default
+  VALIDMAX: 3
+  CATDESC: State - ADC Mode (0=quiet, 1=normal, 2=adcstim, 3=adcThreshold
+  FIELDNAM: ADC Mode (0=quiet, 1=normal, 2=adcstim, 3=adcThreshold)
+  LABLAXIS: ADC mode
+  FORMAT: I1
+
+mode:
+  <<: *support_default
+  VALIDMAX: 3
+  CATDESC: State - Mode (0=Boot, 1=Maintenance, 2=Standby, 3=Science)
+  FIELDNAM: Mode (0=Boot, 1=Maintenance, 2=Standby, 3=Science)
+  LABLAXIS: Mode
+  FORMAT: I1
+
+dyn_thresh_lvl:
+  <<: *support_default
+  VALIDMAX: 3
+  CATDESC: Dynamic Threshold Level (0-3)
+  FIELDNAM: Dynamic Threshold Level (0-3)
+  LABLAXIS: Level
+  FORMAT: I1
+
+num_evnt_last_hk:
+  <<: *support_default
+  VALIDMAX: 262143
+  CATDESC: Number of Events Since Last HK Update
+  FIELDNAM: Number of Events Since Last HK Update
+  LABLAXIS: Num events
+  FORMAT: I6
+
+num_errors:
+  <<: *support_default
+  VALIDMAX: 255
+  CATDESC: Number of Errors
+  FIELDNAM: Number of Errors
+  LABLAXIS: Num errors
+  FORMAT: I3
+
+last_error_num:
+  <<: *support_default
+  VALIDMAX: 255
+  CATDESC: State - Last Error Number
+  FIELDNAM: Last Error Number
+  LABLAXIS: Error num
+  FORMAT: I3
+
+code_checksum:
+  <<: *support_default
+  VALIDMAX: 65535
+  CATDESC: Code Checksum
+  FIELDNAM: Code Checksum
+  LABLAXIS: Checksum
+  FORMAT: I5
+
+spin_period_short:
+  <<: *support_default
+  VALIDMAX: 65535
+  CATDESC: Spin Period Short at T=0
+  FIELDNAM: Spin Period Short at T=0
+  LABLAXIS: Spin period short
+  FORMAT: I5
+
+spin_period_long:
+  <<: *support_default
+  VALIDMAX: 65535
+  CATDESC: Spin Period Long at T=0
+  FIELDNAM: Spin Period Long at T=0
+  LABLAXIS: Spin period long
+  FORMAT: I5
+
+leak_i:
+  <<: *support_default
+  DEPEND_1: adc_channels
+  CATDESC: Leakage Current [I]
+  FIELDNAM: Leakage Current [I]
+  LABLAXIS: Current I
+  labl_ptr: adc_channels
```

Comment:
Maybe I missed an update since I did this, but for the label ptr attribute in leak_i you should have:

```yaml
LABEL_PTR_1: adc_channels_label
```

where adc_channels_label is:
```yaml
adc_channels_label:
  CATDESC: Analog Digital Converter Channels
  FIELDNAM: ADC Channels
  FORMAT: A2
  VAR_TYPE: metadata
```